src/components/ActionModal.js

```diff
@@ -0,0 +1,98 @@
+
+import React from 'react';
+import {
+  StyleSheet, Text, View, TouchableOpacity
+} from 'react-native';
+import PropTypes from 'prop-types';
+import Modal from 'react-native-modal';
+import { PRIMARY_COLOR } from '../constants';
+
+/**
+ * Component that renders a modal with a title, a message and a button to perform an action.
+ *
+ * @example
+ * <ActionModal
+ *  title={this.state.modal.title}
+ *  message={this.state.modal.message}
+ *  button={this.state.modal.button}
+ *  onAction={this.state.modal.onAction}
+ *  onDismiss={this.state.modal.onDismiss}
+ * />
+ *
+ * @param {{ title, message, button, onAction, onDismiss }} props The props of the component
+ * @returns The action modal component to display
+ */
+const ActionModal = (props) => (
```

Comment:
We have a component called `FeedbackModal`, which is used in the whole app. I understand that there is a small difference (the button "I agree") but I don't know if it's needed. To be honest, after the latest wallet update with the terms of service, I feel this modal won't be needed anymore at all, I'm confirming this.

Anyway, there are two clear benefits if we can use the `FeedbackModal` instead of creating this new component, (i) it's easier to maintain the code, (ii) we keep consistency with UI. The latter is the most critical for me because this new modal is a bit different in design compared to the other ones (I think, I didn't check the style but it looked like that in the video).

What do you think?